### PR TITLE
backend: Add AXI Stream as a default configuration

### PR DESCRIPTION
**Pull request by thommythomaso**

### CODE REVIEW

File: jobs/jobs.json [hunk 1]
```diff
@@ -194,6 +194,30 @@
         "testbench": "tb_idma_nd_midend",
         "synth_top": "idma_nd_midend_synth"
     },
+    "rw_axi_rw_axis": {
+        "jobs" : {
+            "simple"          : "backend_rw_axi/simple.txt"
```
Comment (thommythomaso) [inline jobs/jobs.json:199]: @chaoqun-liang, can you maybe add some better test cases here? This is just a placeholder for now.

File: jobs/backend_rw_axi_rw_axis/huge.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+def replace_first_line_in_groups_with_huge(small_modified_path, huge_path, final_output_path):
```
Comment (alex96295) [inline jobs/backend_rw_axi_rw_axis/huge.py:1]: License + autorship

File: jobs/backend_rw_axi_rw_axis/small.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import random
```
Comment (alex96295) [inline jobs/backend_rw_axi_rw_axis/small.py:1]: Ditto

**Review by alex96295 — APPROVED** [review]
Left a couple of small comments, other than that if some test cases exist and CI passes, LGTM

**Pull request merged**